libs/langchain/langchain/vectorstores/elasticsearch.py

```diff
@@ -509,6 +509,7 @@ def __init__(
             ]
         ] = None,
         strategy: BaseRetrievalStrategy = ApproxRetrievalStrategy(),
+        **kwargs: Any,
```

Comment:
I'm a bit worried about using **kwargs in such a large initializer; perhaps we could accept a dict to make it more targeted?

es_params: Optional[Dict[str, Any]] = None

and unpack at destination with **es_params

Reply:
Fixed: https://github.com/langchain-ai/langchain/pull/14435/commits/95e5b51515f9a0e80a61f59bd3630c4727c7c748